admin_tests/preprint_providers/test_views.py

```diff
@@ -192,7 +192,7 @@ def test_html_fields_are_stripped(self):
 
         stripped_advisory_board = '<div><ul><li>Bill Nye</li></ul></div>'
         stripped_description = '<span>Open Preprints Open Science</span>'
-        stripped_footer_links = '<p>Xiv: Support | </p>Contact | <a href=""><span class="fa fa-facebook"></span></a><p></p>'
+        stripped_footer_links = '<p>Xiv: Support | Contact | <a href=""><span class="fa fa-facebook"></span></a></p>'
```

Comment:
They fixed this bug in the new version of bleach, I assume this is just a bad test. It makes no sense as deliberate.